org.uqbar.project.wollok.lib/src/wollok.wlk

```diff
@@ -407,6 +464,26 @@ package lang {
 		override method asSet() = self
 
 		override method anyOne() native
+
+		/**
+		 * Returns a new Set with the elements of both self and another collection.
+		 * @returns a Set
+		 */
+		 method union(another) = self + another
```

Comment:
I am not sure but maybe + is also implemented for lists as concatenation?

On Thu, Jun 16, 2016 at 4:16 PM, Matías Freyre <notifications@github.com>
wrote:

> In org.uqbar.project.wollok.lib/src/wollok.wlk
> <https://github.com/uqbar-project/wollok/pull/744#discussion_r67352062>:
>
> > @@ -407,6 +464,26 @@ package lang {
> >  		override method asSet() = self
> >
> >  		override method anyOne() native
> > +
> > +		/**
> > +		 * Returns a new Set with the elements of both self and another collection.
> > +		 * @returns a Set
> > +		 */
> > +		 method union(another) = self + another
>
> Yes, it's a synonym.
> So far Set doesn't have built-in subclasses, so the answer is that it's
> not overriden yet. It might be in the future.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/uqbar-project/wollok/pull/744/files/43910e97fe384ed85998456476fc455b092f32fc#r67352062>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AEa1OQ9xxeyiTRbxaYl5TcBVZvmfq8gKks5qMVrMgaJpZM4I2zBU>
> .
>